traffic-policy/actions/forward-internal/behavior.mdx

```diff
@@ -0,0 +1,7 @@
+## Behavior
+
```

Comment:
we should include more detail about what happens and when. which traffic policy steps run? is looping allowed? do we re-write `Host` and other headers? things like that